CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other
```

Comment:
Is this necessary to all changes? If it's something relatively simple, I'd be OK with just getting a PR.

Reply:
Agree with you, not all changes would need this process. Gonna update the docs to reflect this 👍